package.json

```diff
@@ -723,9 +726,46 @@
             "configurationAttributes": {
                 "launch": {
                     "properties": {
+                        "scriptPath": {
+                            "type": "string",
+                            "descripttion": "%cmake-tools.debugger.scriptPath.description%",
+                            "default": ""
+                        },
+                        "scriptArgs": {
+                            "type": "array",
+                            "items": {
+                                "type": "string"
+                            },
+                            "default": [],
+                            "description": "%cmake-tools.debugger.scriptArgs.description%"
+                        },
+                        "scriptEnv": {
+                            "type": "array",
+                            "items": {
+                                "type": "object",
+                                "properties": {
+                                    "name": {
+                                        "type": "string",
+                                        "description": "%cmake-tools.debugger.name%"
+                                    },
+                                    "value": {
+                                        "type": "string",
```

Comment:
What do you think?

I can see the reason for it, maybe someone wants to ensure that an environment variable isn't set before configuring, or something like that, but I'm not sure how common that would be. 

Couldn't someone just set it to an empty string?

Reply:
Depending on the platform, empty string and unset may be distinct states for an environment variable. Bash example:

```bash
$ if [[ -z ${foo+x} ]]; then echo true; else echo false; fi
true
$ export foo="bar"
$ if [[ -z ${foo+x} ]]; then echo true; else echo false; fi
false
$ export foo=""
$ if [[ -z ${foo+x} ]]; then echo true; else echo false; fi
false
$ unset foo
$ if [[ -z ${foo+x} ]]; then echo true; else echo false; fi
true
```

Reply:
I think that's a good idea, but I think it may require a couple of changes elsewhere in the code, so we'll leave it for either a different PR or if users request it.